npm/prepublishOnly.js

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env node
+// ---------------------------------------------------------------------------------------------------------------------
+// This script is intended to create man page for this module
+// ---------------------------------------------------------------------------------------------------------------------
+
+require('shelljs/global');
+
+var fs = require('fs'),
+    path = require('path'),
+
+    readme = fs.readFileSync(path.join(__dirname, '../README.md'), 'utf8'),
+    pkg = require(path.join(__dirname, '../package.json')),
+
+    colors = require('colors/safe'),
+    readmeToManPage = require('readme-to-man-page'),
+
+    manPath,
+    manPageContent;
+
+manPath = path.resolve(path.join(__dirname, '../man/newman.1'));
+
+module.exports = function (exit) {
+
+    // create the man page content from markdown
+    manPageContent = readmeToManPage(readme, {
+        name: pkg.name,
+        version: pkg.version,
+        description: pkg.description,
+        section: '1',
+        manual: 'Newman User Manual'
+    });
+
+    // write the generated content to the specified path
+    fs.writeFile(manPath, manPageContent, function (err) {
```

Comment:
This will fail if the `man` subdirectory does not already exist, add a sanity check before this.